### DISCUSSION TITLE
Planned release timeline for Sequelize version 8

### DISCUSSION THREAD

**Discussion by clamx4** [sequelize/sequelize#16966] [Help & Questions] [answered]
Hi,

Is there a planned or estimated release timeline for sequelize v8?

I'm interested in contributing to Sequelize by adding HANA dialect support.
In discussion https://github.com/sequelize/sequelize/discussions/16939, I was recommended making a fork and adding support for a new dialect in there and wait for sequelize v8, which is expected to introduce a plugin system for supporting dialects.

If there is a planned timeline for the release of Sequelize v8, it would be greatly appreciated. This information will help me organize my time more effectively for the contribution, especially considering the effort it might take to maintain an unmerged PR over an extended period. If sequelize v8 takes one or two years to be released, I may consider adding the dialect after the v8 release.

**Comment by WikiRik** [comment, marked as answer]
We don't commit to any timelines since we know that life can easily change with our smaller set of maintainers. But I do expect that v8 is at least one year away.

**Reply by clamx4** [reply]
Thank you, WikiRik.
I will then reconsider when to start adding the HANA dialect.
